crates/sparrow-compiler/src/ast_to_dfg.rs

```diff
@@ -95,6 +95,73 @@ pub(super) fn ast_to_dfg(
     )
 }
 
+pub fn add_udf_to_dfg(
+    udf: Arc<dyn Udf>,
+    dfg: &mut Dfg,
+    arguments: Resolved<Located<AstDfgRef>>,
+    data_context: &mut DataContext,
+    diagnostics: &mut DiagnosticCollector<'_>,
+) -> anyhow::Result<AstDfgRef> {
+    let argument_types = arguments.transform(|i| i.with_value(i.value_type().clone()));
+    let signature = udf.signature();
+
+    // TODO: No locations anymore
+    let fake_location = Located::new("fake".to_owned(), Location::internal_str("fake"));
```

Comment:
Okay for now?

Reply:
`Located::builder()` is used elsewhere, and exists for this case. Possibly pass the location in so that the session code is what decides to use that "fake" location?